pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/model/PanDomainAuthSettings.scala

```diff
@@ -31,8 +32,16 @@ case class Google2FAGroupSettings(
 object PanDomainAuthSettings{
 
   def apply(settingMap: Map[String, String]): PanDomainAuthSettings = {
+    val cookieSettings = CookieSettings(
+      cookieName = settingMap("cookieName"),
+      assymCookieName = settingMap("assymCookieName")
+    )
 
-    val googleAuthSettings = GoogleAuthSettings(settingMap("googleAuthClientId"), settingMap("googleAuthSecret"))
+    val oAuthSettings = OAuthSettings(
+      settingMap("clientId"),
+      settingMap("clientSecret"),
+      settingMap("discoveryDocumentUrl")
```

Comment:
what would `discoveryDocumentUrl` be in a google auth context?

Reply:
`https://accounts.google.com/.well-known/openid-configuration` - see the top level PR description

Reply:
![img](https://media.giphy.com/media/EX6wKXVM9IOlO/giphy.gif)